tests/etl/test_perf_data_adapters.py

```diff
@@ -63,16 +63,14 @@ def test_adapt_and_load(self):
                 if talos_datum.get('summary'):
                     # if we have a summary, ensure the subtest summary values made
                     # it in
-                    for measure in ['min', 'max', 'std', 'mean', 'median']:
-                        self.assertEqual(
-                            round(talos_datum['summary']['subtests'][testname][measure], 2),
-                            perfdata[measure])
+                    self.assertEqual(
+                        round(talos_datum['summary']['subtests'][testname]['filtered'], 2),
+                        perfdata['value'])
                 else:
                     # this is an old style talos blob without a summary. these are going
```

Comment:
Here and on other places which ingest the "legacy" `value` field, do we expect TH to ever bump into a talos logs without a `filtered` field? I.e. right now AFAIK all the logs have `filtered`, so unless there's a good reason to keep them (like maybe the talos logs for the beta branch etc don't have `filtered`), I'd say just remove all these legacy "else" clauses.